### DISCUSSION TITLE
产品最终想做成什么样子？具备哪些功能？

### DISCUSSION THREAD

**Discussion by honghaier250** [usememos/memos#54] [General]
@boojack

**Comment by boojack** [comment]
一个主要的核心点：快速、私有化部署。

其他一些重要点：
* UI 简洁好用；
* 编辑器支持 Markdown 语法；
* 集成 Notion、GitHub gist、微信公众号等常用平台。

**Reply by cicai1631** [reply]
整和到微信小程序或者开发个安卓客户端？

**Reply by boojack** [reply]
> 整和到微信小程序或者开发个安卓客户端？

把网页端做好就行，目前已有其他小伙伴开发的小程序 https://github.com/Rabithua/memos_wmp ，可以自行体验一下

**Comment by monlor** [comment]
可以关闭注册的功能么，仅限个人使用

**Comment by honghaier250** [comment]
功能开发优先级列表是什么样子的呢？
@boojack

**Comment by boojack** [comment]
没有优先级安排，有空时随缘更新;)
